### ISSUE TITLE
Enable / disable data on disk and MerkleDb in dev environments

### ISSUE THREAD

**Issue by artemananiev** [hashgraph/hedera-services#6885]
This task has two parts:

1. In 0.39, change dev env settings (hedera-node/data/config/bootstrap.properties) to whatever we have in testnet / mainnet: JasperDB and only file store and smart contracts store on disk

2. In 0.40, change dev env settings to data on disk + MerkleDb

Item 2 above can be done only after known MerkleDb issues are resolved in `develop` branch.

**Comment by artemananiev** [comment]
0.40 changes are tracked in a separate issue: https://github.com/hashgraph/hedera-services/issues/6944. This one is now scoped to 0.39

**Comment by artemananiev** [comment]
Merged to `release/0.39`
